opp_gui/src/widgets/tool_path_parameters_editor_widget.cpp

```diff
@@ -209,6 +312,65 @@ void ToolPathParametersEditorWidget::onGenerateToolPathsComplete(
   }
 }
 
+
+void ToolPathParametersEditorWidget::onGenerateHeatToolPathsComplete(
+    const actionlib::SimpleClientGoalState& state,
+    const heat_msgs::GenerateHeatToolPathsResultConstPtr& res)
+{
+  for (int i = progress_dialog_->minimum(); i < progress_dialog_->maximum(); ++i)
+  {
+    progress_dialog_->setValue(i);
+    ros::Duration(0.01).sleep();
+  }
+  progress_dialog_->hide();
+
+  if (state != actionlib::SimpleClientGoalState::SUCCEEDED)
+  {
+    std::string message = "Action '" + GENERATE_HEAT_TOOLPATHS_ACTION + "' failed to succeed";
+    QMessageBox::warning(this, "Tool Path Planning Error", QString(message.c_str()));
+  }
+  else
+  {
+    if (!res->success || !res->tool_path_validities[0])
+    {
+      QMessageBox::warning(this, "Heat Tool Path Planning Error", "Heat tool path generation failed");
```

Comment:
Same comment as above

Reply:
@marip8 Please call and we'll discuss the indicated items off line